doc/manual/src/contributing/hacking.md

```diff
@@ -108,3 +106,72 @@ These tests include everything that needs to interact with external services or
 Because these tests are expensive and require more than what the standard github-actions setup provides, they only run on the master branch (on <https://hydra.nixos.org/jobset/nix/master>).
 
 You can run them manually with `nix build .#hydraJobs.tests.{testName}` or `nix-build -A hydraJobs.tests.{testName}`
+
+### Installer tests
+
+With just a little one-time setup, the Nix repository's GitHub Actions continuous integration (CI) workflow can easily test the installer each time you push to a branch.
```

Comment:
I would abstain from value judgements. What may be a little
or easy for you may be overwhelming for others.

 ```suggestion
After a one-time setup, the Nix repository's GitHub Actions continuous integration (CI) workflow can test the installer each time you push to a branch.
```

Reply:
This isn't a value judgement about how easy it is for _me_, it's a value judgement about how manually building your own installer and testing it is an exercise in masochism. I think it's important to invite people who've been scarred by how hard this has been historically to give the automated workflow a try, but I'd rather move on at this point.